fix(github-actions): add missing workflow age field to all rendered components (#4762)
backstage/community-plugins: workspaces/github-actions/.changeset/rich-apricots-smash.md

---
'@backstage-community/plugin-github-actions': patch
---

> Add missing workflow age field to provide equivalent information to azure devops plugin. Without this there is no way to tell when the workflow ran without navigating away to github itself.

Reviewer: ```suggestion
Add workflow age field to help indicate when the workflow last ran
```

As these get [rolled up into changelogs](https://github.com/backstage/community-plugins/blob/main/workspaces/github-actions/plugins/github-actions/CHANGELOG.md) this should be written with that context in mind.
